api/app/routes/admin/faqs.py

```
@@ -94,6 +94,54 @@ async def update_existing_faq_route(faq_id: str, faq_item_update: FAQItem):
+    Returns:
+        The updated FAQ with the new verification status
+    """
+    logger.info(f"Admin request to set FAQ {faq_id} verification to: {verified}")
+
+    try:
+        # Get the current FAQ
+        all_faqs = faq_service.get_all_faqs()
+        current_faq = next((faq for faq in all_faqs if faq.id == faq_id), None)
+
+        if not current_faq:
+            raise FAQNotFoundError(faq_id)
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)